main/RFXNames.h

```diff
@@ -57,7 +57,21 @@ enum _eTimerType
 	TTYPE_MONTHLY_WD,
 	TTYPE_YEARLY,
 	TTYPE_YEARLY_WD,
-	TTYPE_END
+	TTYPE_END,
```

Comment:
TTYPE_END should be at the 'end'

Reply:
Thanks. That has been taken care of now.